scss/_patterns_lists.scss

```diff
@@ -318,6 +318,9 @@ $list-step-bullet-margin: $sph--x-large;
 
   .p-stepped-list__title + .p-stepped-list__content {
     margin-left: $bullet-width-mobile + $list-step-bullet-margin;
+    @media (min-width: $threshold-6-12-col) {
+      padding-top: $sp-unit;
```

Comment:
shouldn't this use the paragraph nudge value?

Reply:
@lyubomir-popov revised to use `$spv-nudge`